content/en/blog/_posts/2023-04-14-improving-support-go-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Improving Support: Go Updates"
+date: 2023-04-14
```

Comment:
Do we want this to publish before or after the v1.27 release? I'm minded to suggest publication this week.

/hold
pending assignment of a publication date (we need to check with release comms if this comes out in the post release period)

Reply:
fine with me either way, it's not actually tied to the 1.27 release cycle